### DISCUSSION TITLE
Two muscians: Two sends with one Jamulus client

### DISCUSSION THREAD

**Discussion by joehai** [jamulussoftware/jamulus#2141] [General]
Hi, 
we're using Jamulus for rehearsal. Vocal and Bass are located in one room and feed a raspberry with Jamulus 3.8.0 having a analogue mixer before the raspberry (stereo in/stereo out soundcard).
I'd like to send two seperate channels (one for vocal, one for bass) to the server, so that all musicians can mix bass and vocal seperatly with dedicated faders.
Is there a way on doing this without "misusing" the stereo (so sending bass on left & vocs on right) sending feature?  
Best regards & thanks! Jörg

**Comment by pljones** [comment]
Why not run two clients?

**Reply by gilgongo** [reply]
Two clients is probably the best way. See also the Save Mixer Channels function, which might come in handy: https://jamulus.io/wiki/Software-Manual#file--loadsave-mixer-channels-setup

**Reply by joehai** [reply]
Is there a manual how to setup/run two Jamulus clients (which use one audio hardware and -if possible- to have one mix for the room with two musicians)?

**Reply by gilgongo** [reply]
Depending on the platform, you should just be able to start each client, open Settings and connect the inputs to the relevant device you want to use.

"one mix for the room with two musicians" should also be possible as long as you don't violate [The Golden Rule](https://jamulus.io/wiki/FAQ#why-should-i-not-listen-to-my-own-signal) though :-)

**Reply by pljones** [reply]
You need to run each client on a separate port (`-p 50000` for one, `-p 50001` for the other, or something).

Are you running a full Linux edition on the Pi?  If so, which one?  How are you running the client, currently?

**Reply by joehai** [reply]
I'm running the Raspberry PI 4 Jambox Edition (https://github.com/kdoren/jambox-pi-gen) which also comes with an Web-UI solution.  Jamulus version is 3.8.0.

**Reply by pljones** [reply]
Have you tried asking over at https://github.com/kdoren/jambox-pi-gen/discussions by the way?

@kdoren - can you answer this?

(I don't know which desktop it is or how the desktop icon runs the Jamulus client.)

**Comment by chrisrimple** [comment]
I talk about multi-client configurations in this doc...

https://docs.google.com/document/d/1smcvsxdaaViPQvGMQHmah_6BQeqowhmGSFMHfnlY2FI/
